project1/styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+    background-color: rgba(236, 236, 236, 0.226)
+}
+
+main {
+    background-color: rgb(255, 255, 255);
+    padding-top: 10%;
+    padding-left: 25%;
+    padding-right: 25%;
+    padding-bottom: 25%;
+
+}
+
+p {
```

Comment:
As a final clean up of your CSS, you want to
1. alphabetize your declarations by property
2. remove any extra whitespace/linebreaks inside the brackets

So, the ruleset for your `p` would look like:

```css
p {
    color: rgba(83, 83, 83, 0.801);
    font-family: 'Lato', sans-serif;
    line-height: 150%;
    text-align: justify;
}
```